recipes/sqlpp11/all/conanfile.py

```diff
@@ -12,60 +16,56 @@ class Sqlpp11Conan(ConanFile):
     homepage = "https://github.com/rbock/sqlpp11"
     description = "A type safe SQL template library for C++"
     topics = ("sql", "dsl", "embedded", "data-base")
+    package_type = "header-library"
     settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
 
     @property
-    def _min_stdcpp_version(self):
-        return 11 if tools.Version(self.version) < "0.61" else 14
+    def _min_cppstd(self):
+        return 11 if Version(self.version) < "0.61" else 14
 
     @property
     def _compilers_minimum_version(self):
         return {
             "gcc": "5",
             "Visual Studio": "14",
+            "msvc": "14.1",
```

Comment:
I might be missunderstanding this, but isn't the proper msvc version 190 in this case?

```suggestion
            "msvc": "190",
```